pom.xml

```diff
@@ -114,24 +117,22 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <configuration>
                         <encoding>utf-8</encoding>
-                        <source>1.6</source>
-                        <target>1.6</target>
+                        <source>1.8</source>
+                        <target>1.8</target>
```

Comment:
Do you have to go to 1.8? what's the lowest can you go?

Reply:
We can actually do 1.6. 1.6 wasn't working in intellij, but I think that's because of my environment. I'm pushing a commit to go back to 1.6 now.